### ISSUE TITLE
Add support for HTTP(S) Graylog endpoints

### ISSUE THREAD

**Issue by mersadk** [mattwcole/gelf-extensions-logging#13]
Hi,

From what I can see this only supports UDP endpoints. It would be nice to have support for HTTP(S) endpoints as well.
If you are willing to merge it, I can create pull request for that.

**Comment by mattwcole** [comment]
I think this would be a great addition. A new implementation of [`IGelfClient`](https://github.com/mattwcole/gelf-extensions-logging/blob/dev/src/Gelf.Extensions.Logging/IGelfClient.cs) should do the job. Feel free to put a PR up, however I've got a small tweak to make to the dev branch after the last PR was merged that should be done first. I will be able to push it tonight when I am home from work.

**Comment by mersadk** [comment]
Great, just let me know when you're done and I'll create PR.

**Comment by mersadk** [comment]
Wrong button, sorry :/

**Comment by mattwcole** [comment]
All done!

**Comment by mersadk** [comment]
I've created pull request.

I've added HttpGelfClient and modified unit tests to test both UDP and HTTP. I didn't update readme file.

To use HTTP/HTTPS host in settings needs to start with http://... and it will recognize it needs to use HttpGelfClient. If Graylog input is running on some custom port, port should be included in host name with ```:{port}```. Stand-alone port setting is ignored for HTTP.
